### DISCUSSION TITLE
why erlang 24  can not  install rabbitmq-server-3.8.16-1.el8.noarch.rpm &rabbitmq-server-3.8.15-1.el8.noarch.rpm?

### DISCUSSION THREAD

**Discussion by shengyingwenrou** [rabbitmq/rabbitmq-server#3094] [Other]
Why erlang 24  can not  install rabbitmq-server-3.8.16-1.el8.noarch.rpm &rabbitmq-server-3.8.15-1.el8.noarch.rpm?
   still tips:
   error: Failed dependencies:
	erlang >= 23.2 is needed by rabbitmq-server-3.8.17-1.el8.noarch

**Comment by shengyingwenrou** [comment]
I solve it by  force install
rpm -ivh --nodeps --force rabbitmq-server-3.7.4-1.el7.noarch.rpm
then
vi /usr/lib/rabbitmq/bin/rabbitmq-server
add content 
ERLANG_HOME=/usr/local/erlang
export PATH=$PATH:$ERLANG_HOME/bin

mark it.

**Comment by michaelklishin** [comment]
There a lot of information in this question to offer an informed opinion. Consider sharing **a complete snippet** of what you are doing.

According to the message, my best guess is that had a different Erlang version installed. Erlang version requirements are [correctly set in the RPM spec](https://github.com/rabbitmq/rabbitmq-packaging/blob/v3.8.x/RPMS/Fedora/rabbitmq-server.spec#L2).
